### PR TITLE
Refactor for Alexa Hosted

### PR DESCRIPTION
**Pull request by robm26**
Refactor for Alexa Hosted

### CODE REVIEW

**Comment by franklin-lobb** [comment]
Mike - I've reviewed these changes and they look good to me.  Can you take a look?  Rob and I discussed and would like to drop the publish page from the flow and just end on customize.  If you agree, one of us (or you) can drop the button and the page.

File: models/en-US.json [hunk 1]
```diff
@@ -4,21 +4,34 @@
...
-                    "name": "AMAZON.CancelIntent",
+                    "name": "HelloIntent",
+                    "slots": [],
                     "samples": [
-                        "cancel"
+                        "hello"
                     ]
                 },
+                {
+                    "name": "AMAZON.FallbackIntent",
```
Comment (mdinescu) [inline models/en-US.json]: doesn't this require some code changes to support the fall back intent?

Comment (mdinescu) [inline models/en-US.json]: doesn't this require some code changes, to support the FallbackIntent in the code?

File: instructions/create-aws-hosted-function.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Build An AWS-Hosted Hello Buttons Skill
```
Comment (mdinescu) [inline instructions/create-aws-hosted-function.md]: is `AWS-Hosted` the official name we're going with "classic" skills?  It seems like the Alexa-hosted and AWS-hosted are essentially both hosted on AWS. I wonder if we should say "Build a Hello Buttons Skill Hosted in Your Own AWS Account"

File: instructions/cli.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Build An Alexa Hello Buttons Skill
```
Comment (mdinescu) [inline instructions/cli.md]: nit: should this be "Build an Alexa Hello Buttons Skill Using the CLI" !?

File: lambda/custom/index.js [hunk 4]
```diff
@@ -146,6 +147,24 @@ const main = {
       return response;  
     }  
   },
+    FallbackHandler: {
```
Comment (mdinescu) [inline lambda/custom/index.js]: there doesn't seem to be a difference between the FallbackHandler and the HelpIntentHandler: why not use the same handler to handle both intents?

**Comment by robm26** [comment]
Updated Instructions

**Pull request closed without merging**